src/panels/GeoLocationPanel/Search/SearchOverlay.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Overlay, Transition } from '@mantine/core';
+import { useClickOutside } from '@mantine/hooks';
+
+import { GeoSearch } from './GeoSearch';
+
+export function SearchOverlay({
+  search,
+  visible,
+  close,
+  h,
+  setMouseMarker,
+  setCoordinates,
+  setCustomName
+}: {
+  search: string;
+  h: number;
+  visible: boolean;
+  close: () => void;
+  setMouseMarker: ({ x, y }: { x: number; y: number }) => void;
+  setCoordinates: (coordinates: { lat: number; long: number; alt: number }) => void;
+  setCustomName: (name: string) => void;
+}) {
```

Comment:
```suggestion
interface Props {
  search: string;
  h: number;
  visible: boolean;
  close: () => void;
  setMouseMarker: ({ x, y }: { x: number; y: number }) => void;
  setCoordinates: (coordinates: { lat: number; long: number; alt: number }) => void;
  setCustomName: (name: string) => void;
}

export function SearchOverlay({
  search,
  visible,
  close,
  h,
  setMouseMarker,
  setCoordinates,
  setCustomName
}: Props) {
```

Or, considering how many times it's now done this way, have you guys made some code style decision related to this that has not been communicated?